fiftyone/utils/torch.py

```
@@ -2605,6 +2667,244 @@ def _parse_inputs(
+
+        Args:
+            worker_id: the worker ID
+        """
+        import fiftyone.core.odm.database as food
+
+        # Ensure that each process creates its own MongoDB clients
+        # https://pymongo.readthedocs.io/en/stable/faq.html#using-pymongo-with-multiprocessing
+        # pylint:disable-next=protected-access
+        food._disconnect()
+
+    def _get_decoder_for_video(self, video_path, frame_range=None):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_get_decoder_for_video`

(ANN202)